articles/active-directory/develop/quickstarts/active-directory-aspnetwebapp.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Azure AD v2 ASP.NET web server quickstart | Microsoft Docs
+description: Learn how to implement Microsoft Sign-In on an ASP.NET web app using OpenID Connect
+services: active-directory
+documentationcenter: dev-center-name
+author: andretms
+manager: mtillman
+editor: ''
+
+ms.assetid: 820acdb7-d316-4c3b-8de9-79df48ba3b06
+ms.service: active-directory
+ms.devlang: na
+ms.topic: article
+ms.tgt_pltfrm: na
+ms.workload: identity
+ms.date: 07/15/2018
+ms.author: andret
+ms.custom: aaddev
+
+---
+
+# Add sign-in with Microsoft to an ASP.NET web app
+
+In this quickstart, you'll learn how a ASP.NET Web App can sign in personal accounts (hotmail.com, outlook.com, others) and work and school accounts from any Azure Active Directory instance.
+
+![How the sample app generated by this quickstart works](media/active-directory-aspnetwebapp/aspnetwebapp-intro.png)
+
+> [!div renderon="docs"]
+> ## Register your application and download your quickstart app
+> You have two options to start your quickstart application:
+> * [Express] [Option 1: Register and auto configure your app and then download your code sample](#option-1-register-and-auto-configure-your-app-and-then-download-your-code-sample)
+> * [Manual] [Option 2: Register and manually configure your application and code sample](#option-2-register-and-manually-configure-your-application-and-code-sample)
+>
+> ### Option 1: Register and auto configure your app and then download your code sample
+>
+> 1. Go to the [Azure portal - Application Registration](https://portal.azure.com/signin/index/?Microsoft_AAD_RegisteredApps=true#blade/Microsoft_AAD_RegisteredApps/applicationsListBlade/quickStartType/AspNetWebAppQuickstartPage/sourceType/docs).
```

Comment:
The APEX doc team doesn't advise it, but I've used it in the past in other doc teams. The format for opening a new tab for a link is to do it like you would in HTML, so it would be **<a href="http://url_goes_here.com" target="_blank">Link text goes here</a>**. In this case, it would be <a href="https://portal.azure.com/signin/index/?Microsoft_AAD_RegisteredApps=true#blade/Microsoft_AAD_RegisteredApps/applicationsListBlade/quickStartType/AspNetWebAppQuickstartPage/sourceType/docs" target="_blank">Azure portal - Application Registration</a>. It's the **target="_blank"** that opens a new tab or window depending on the browser or how it's set to interpret that.